src/status_im2/contexts/onboarding/create_profile/view.cljs

```diff
@@ -48,19 +49,34 @@
     :else                        nil))
 
 (defn button-container
-  [keyboard-shown? children]
+  [keyboard-shown? background? children]
   [rn/view {:style {:margin-top :auto}}
    (if keyboard-shown?
-     [blur/ios-view
-      {:blur-amount      34
-       :blur-type        :transparent
-       :overlay-color    :transparent
-       :background-color (if platform/android? colors/neutral-100 colors/neutral-80-opa-1-blur)
-       :style            style/blur-button-container}
-      children]
+     (if background?
+       [blur/ios-view
+        (merge
+         {:blur-amount      34
+          :blur-type        :transparent
+          :overlay-color    :transparent
+          :background-color (if platform/android? colors/neutral-100 colors/neutral-80-opa-1-blur)
+          :style            style/blur-button-container})
+        children]
+       [rn/view
+        (merge
+         {:style style/blur-button-container})
+        children])
      [rn/view {:style style/view-button-container}
       children])])
 
+(defn show-button-background
+  [keyboard-shown?]
+  (let [{:keys [keyboard-height]} (hooks/use-keyboard)
+        window-height             (rf/sub [:dimensions/window-height])
+        remaining-screen-height   (- window-height keyboard-height)]
```

Comment:
should this not include the height of the button container? 
```
(- window-height keyboard-height button-container)]
```
because that would be the marking of the line

Reply:
yeah this seems to be why in some devices its not working as expected